pkg/cmd/server/origin/master.go

```diff
@@ -447,6 +465,21 @@ func (c *MasterConfig) getRequestContextMapper() kapi.RequestContextMapper {
 	return c.RequestContextMapper
 }
 
+// ensureMasterAuthorizationNamespace is called as part of global policy initialization to ensure master namespace exists
+func (c *MasterConfig) ensureMasterAuthorizationNamespace() {
+
+	// ensure that master namespace actually exists
+	namespace, err := c.KubeClient().Namespaces().Get(c.MasterAuthorizationNamespace)
+	if err != nil {
+		namespace = &kapi.Namespace{ObjectMeta: kapi.ObjectMeta{Name: c.MasterAuthorizationNamespace}}
+		kapi.FillObjectMetaSystemFields(api.NewContext(), &namespace.ObjectMeta)
+		namespace, err = c.KubeClient().Namespaces().Create(namespace)
```

Comment:
if you are not using the 'namespace' var here, then you can just silent it, right?